django/contrib/gis/utils/layermapping.py

```diff
@@ -339,7 +340,7 @@ def check_unique(self, unique):
                 "Unique keyword argument must be set with a tuple, list, or string."
             )
 
-    # Keyword argument retrieval routines ####
+    # #### Keyword argument retrieval routines
```

Comment:
Changed for consistency with other similar comments that were also getting edited